test/basic_tests.jl

```diff
@@ -87,7 +87,7 @@ LGBM_PATH = if isabspath(LGBM_PATH) LGBM_PATH else abspath(joinpath(pwd(), "..",
     # Not the case while the random functions are enabled: bagging fraction and feature fraction
     # but they should generally be otherwise
     @test_broken isapprox(p_binary, p_multi[:, 2], rtol=1e-16)
```

Comment:
I still can't see how the below works but these probabilities aren't similar. Is the order in p_multi the right way, i.e. is p_multi corresponding to probabilitiesfor 0, and 1 at indices 1,2 respectively? Is it reversed? Is it possibly random on the basis of the order of the occurrences in the training data?

Reply:
ok for this one, what would be a reasonably acceptable tolerance? I looked into the outputs and we're talking about the following: 
![image](https://user-images.githubusercontent.com/59925069/98245929-89459d00-1f69-11eb-997c-671576bba3bc.png)

I was very generous with the tolerance...

Reply:
I think you should look at statistics of this ... like is the difference nearly constant? does the direction vary (sometimes negative, sometime positive) etc. Because given the predictions are the same but probabilities are changed by 1%ish it looks like it could be a constant bias or something similar ...